### ISSUE TITLE
Favicon gathering doesn't load the entire (3rd+) level domain, only the 2nd

### ISSUE THREAD

**Issue by luponata** [bitwarden/web#1271]
### Steps To Reproduce

1. Create a new Login item
2. Fill the new item with your credentials and with a proper URL

### Expected Result

I'm expecting Bitwarden to load and save the relative domain favicon

### Actual Result

Only the 2nd level domain favicon get loaded

### Additional Context

So if i have multiple 3rd level domain applications as shown below:

Real world:
- example.com (Main site - Company Favicon)
- crm.example.com (CRM site - CRM Favicon)
- webmail.example.com (Webmail - Exchange OWA Favicon)

If i create those 3 items, all of them are going to get saved with the main site favicon, despite reaching each website via browser, the correct respective icon will be shown

Bitwarden items:
- example.com (rightly shows the main site favicon)
- crm.example.com (Shows the main site favicon instead of the CRM one)
- webmail.example.com (Shows the main site favicon instead of the Exchange one)

### Operating System

Windows, Linux

### Web Browser

Chrome, Firefox, Opera

### Build Version

2.24.1

**Comment by chao813** [comment]
@luponata Do you have an example of a site that has multiple flavicons on each domain level?

**Comment by luponata** [comment]
Yes, every company that work with (hosted) software made by other people.

Imagine if you use a Zimbra server as a mail server, salesforce as a PMS and so on...

Every URL is going to have different favicon: 

- zimbra.company.com
- salesforce.company.com

But on Bitwarden they are going to have the same favicon as the main domain "company.com"

On the Bitwarden extension, you are able to configure the "Default URI Match Detection" so when you reach your zimbra server, the extension is not going to show you all the saved passwords for *.company.com but only for the exact domain. 

The same concept should be applied to the favicon manager

**Comment by luponata** [comment]
The issue is still unsolved :(

**Comment by dwbit** [comment]
Hey Luponata, since this is a discussion/request, the best place to continue the chat is on the [Bitwarden Community Forums](https://community.bitwarden.com/). I'll close this issue now.
